src/apps/users/domain.py

```diff
@@ -98,6 +101,9 @@ def from_user(cls, user: User) -> "PublicUser":
             id=user.id,
         )
 
+    def get_full_name(self) -> str:
```

Comment:
Please add a unit test. Looks like it goes here `src/apps/users/tests/unit/test_user_domain.py`

Eg

```
def test_user_get_full_name():
    user = domain.User(first_name="John", last_name="Doe")
    assert user.get_full_name() == "John Doe"

def test_user_get_full_name_with_no_last_name():
    user = domain.User(first_name="John")
    assert user.get_full_name() == "John"
```